src/views/Home/index.tsx

```diff
@@ -8,18 +15,83 @@ import {
 import { useTranslation } from '@ifrc-go/ui/hooks';
 
 import Page from '#components/Page';
+import {
+    AlertEnumsQuery,
+    AlertEnumsQueryVariables,
+    CountryListQuery,
+    CountryListQueryVariables,
+} from '#generated/types/graphql';
+import useInputState from '#hooks/useInputState';
 
 import AlertsTable from './AlertsTable';
 import AlertsView from './AlertsView';
+import Filters, { FilterValue } from './Filters';
 
 import i18n from './i18n.json';
 import styles from './styles.module.css';
 
+const ALERT_ENUMS = gql`
+query AlertEnums {
+    enums {
+      AlertInfoCertainty {
+        key
+        label
+      }
+      AlertInfoUrgency {
+        label
+        key
+      }
+      AlertInfoSeverity {
+        key
+        label
+      }
+    }
+}`;
+
+// NOTE: alertFilters is related with filteredAlertCount
+const COUNTRIES_LIST = gql`
+query CountryList {
+  public {
+    allCountries(alertFilters: {}) {
+      name
+      id
+      iso3
+      filteredAlertCount
+    }
+  }
+}
+`;
+
+const defaultFilterValue: FilterValue = {
+    countries: [],
+    regions: [],
```

Comment:
Should we allow multiple countries and multiple regions filter? @thenav56 @frozenhelium @samshara

Reply:
@barshathakuri Why was this conversation resolved? Did we change them to single select?